pandas/_libs/tslibs/period.pyx

```diff
@@ -1463,6 +1463,45 @@ cdef class _Period(object):
 
     @property
     def qyear(self):
+        """
+        Get the fiscal year component of the Period.
```

Comment:
Find the fiscal year the Period lies in according to the specified starting-quarter.